src/components/icons/index.ts

```diff
@@ -3,6 +3,9 @@ import Dashboard from "./assets/dashboard.svg";
 import Order from "./assets/order.svg";
 import Menu from "./assets/menus.svg";
 import Review from "./assets/review.svg";
+/* 위에도 추후 아래처럼 변경 예정 */
```

Comment:
TODO 주석으로 표현해주시면 나중에 해야되는 작업들을 찾을 때 TODO 키워드로 찾을 수 있어서 조금 더 편리할 것 같아요!
```suggestion
/* TODO: 위에도 추후 아래처럼 변경 예정 */
```

Reply:
@Seung-wan 넵!